sklearnex/tests/test_patching.py

```diff
@@ -56,18 +56,24 @@
 
 
 @pytest.mark.parametrize("dtype", DTYPES)
-@pytest.mark.parametrize(
-    "dataframe, queue", get_dataframes_and_queues(dataframe_filter_="numpy")
-)
+@pytest.mark.parametrize("dataframe, queue", get_dataframes_and_queues())
 @pytest.mark.parametrize("metric", ["cosine", "correlation"])
 def test_pairwise_distances_patching(caplog, dataframe, queue, dtype, metric):
     with caplog.at_level(logging.WARNING, logger="sklearnex"):
+        if dtype == np.float16 and queue and not queue.sycl_device.has_aspect_fp16:
+            pytest.skip("Hardware does not support fp16 SYCL testing")
```

Comment:
Would we want a similar check for fp64 support?

Reply:
So far CI testing has shown no issues with fp64 support, but if this is something popping up on windows we can look at running special CI jobs against this PR and make requisite changes for it.

Reply:
There wouldnt be an issue on PVC since its supporting, but I am guessing there may be on DG1 and DG2. Will keep an eye out in the future and can add as needed.